op-chain-ops/genesis/db_migration.go

```diff
@@ -21,7 +21,7 @@ import (
 
 var (
 	abiTrue                         = common.Hash{31: 0x01}
-	bedrockTransitionBlockExtraData = []byte("BEDROCK")
+	BedrockTransitionBlockExtraData = []byte("BEDROCK")
```

Comment:
If we're making it public and mutable maybe it's worth a comment that this must not be larger than 32 bytes (because of post-Merge consensus-layer block encoding)